lib/workers/repository/init/merge.ts

```diff
@@ -27,16 +27,21 @@ export async function detectRepoFileConfig(): Promise<RepoFileConfig> {
   const cache = getCache();
   let { configFileName } = cache;
   if (configFileName) {
-    let configFileParsed = (await platform.getJsonFile(configFileName))!;
+    let configFileRaw: string | undefined = (await platform.getRawFile(
```

Comment:
```suggestion
    let configFileRaw = (await platform.getRawFile(
```
don't cast to non-null and back